### PR TITLE
Move some branch steps to stabliziation week

### PR DESCRIPTION
**Pull request by ehelms**
The goal here is to identify as many things that can be either done and merged or a PR staged for prior to branch day.

### CODE REVIEW

File: procedures/foreman/branch.md.erb [hunk 1]
```diff
@@ -56,19 +56,31 @@
...
+    ```
+    scp mash_scripts/foreman/<%= release %>.0/*.mash root@koji.katello.org:/etc/mash/
+    ```
+- [ ] Update mash script if needed - [collection-mash-split.py](https://github.com/theforeman/foreman-infra/blob/master/koji/collection-mash-split.py)
+- [ ] Add the new release to the JJB job definitions in [foreman-infra](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/)
+  - [ ] Create a `<%= release %>.groovy` in `pipelines/vars/foreman/` describing the operating systems the release supports
+  - [ ] Add `<%= release %>` to the version list in `theforeman.org/yaml/includes/foreman_versions.yaml.inc`
+  - [ ] Add `<%= release %>` to the version list in `centos.org/jobs/foreman-pipelines.yml`
+  - [ ] Create `test_<%= release.tr('.', '_') %>_stable.yaml` and `test_proxy_<%= release.tr('.', '_') %>_stable.yaml` in `yaml/jobs/`
+- [ ] Open PR to remove the oldest version to keep last 3 from the files/directories in the previous steps
```
Comment (ekohl) [inline procedures/foreman/branch.md.erb:76]: I think we should rephrase this. At the time of branching, we can assume the previous version is GA. What we want is the supported versions (at the time of writing 2.0 and 2.1), the next version (2.2) and nightly.

Would it be better to suggest dropping all EOL versions?

Comment (ehelms) [reply procedures/foreman/branch.md.erb]: Hows this new wording?

File: procedures/foreman/branch.md.erb [hunk 2]
```diff
@@ -56,19 +56,31 @@
...
+    scp mash_scripts/foreman/<%= release %>.0/*.mash root@koji.katello.org:/etc/mash/
+    ```
+- [ ] Update mash script if needed - [collection-mash-split.py](https://github.com/theforeman/foreman-infra/blob/master/koji/collection-mash-split.py)
+- [ ] Add the new release to the JJB job definitions in [foreman-infra](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/)
+  - [ ] Create a `<%= release %>.groovy` in `pipelines/vars/foreman/` describing the operating systems the release supports
+  - [ ] Add `<%= release %>` to the version list in `theforeman.org/yaml/includes/foreman_versions.yaml.inc`
+  - [ ] Add `<%= release %>` to the version list in `centos.org/jobs/foreman-pipelines.yml`
+  - [ ] Create `test_<%= release.tr('.', '_') %>_stable.yaml` and `test_proxy_<%= release.tr('.', '_') %>_stable.yaml` in `yaml/jobs/`
+- [ ] Open PR to remove any jobs that are related to end of life versions
+- [ ] Open PR to add <%= release %> to [Forklift versions config](https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml)
```
Comment (ekohl) [inline procedures/foreman/branch.md.erb:77]: So we noticed that merging this PR breaks the upgrade pipelines. Do we want to note anything about that it should only be merged after packaging branching has happened?

Comment (tbrisker) [reply procedures/foreman/branch.md.erb:77]: How about this @ekohl ?
```suggestion
- [ ] Open PR to add <%= release %> to [Forklift versions config](https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml). Once the PR is merged, upgrade pipelines will fail, so do not merge it before packaging has been branched.
```

Comment (ekohl) [reply procedures/foreman/branch.md.erb:77]: Sounds good to me.

**Comment by tbrisker** [comment]
Thanks @ehelms and @ekohl !

**Pull request merged**